ml-agents/mlagents/trainers/action_info.py

```diff
@@ -8,3 +8,4 @@ class ActionInfo(NamedTuple):
     action: Any
     value: Any
     outputs: ActionInfoOutputs
+    agents: List[str]
```

Comment:
agent_ids?

Reply:
I know we use `agents` to refer to lists of IDs elsewhere, but I don't like that either :)

Reply:
[Maybe out of scope here] 
agent ids are sometimes int and sometimes string, maybe we should create a custom type Union[int,str] ? (unless at this stage of the code, agent ids are always strings)

Reply:
Changed to `agent_ids`. Unfortunately we're using `agent_id` in BatchedStepResult...